src/sql/sqlops.proposed.d.ts

```diff
@@ -697,4 +697,35 @@ declare module 'sqlops' {
 		 */
 		readonly retainContextWhenHidden?: boolean;
 	}
+
+	export enum DataProviderType {
```

Comment:
it the long run we should find a way to avoid having the core app know about all the extensions.